aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Server.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Components.Server">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides classes for configuring circuits and authentication state in Blazor Server applications. Not intended for use outside the Blazor Server framework.</summary>
```

Comment:
This looks incorrect.

> Provides classes for operating a Blazor Server applications. This namespace includes types such as CircuitOptions that allows fine-grained control of Blazor Server.

Reply:
Hm. Which part is incorrect? I edited the wording here a bit. Let me know what you think.